src/main/java/com/spikes2212/command/genericsubsystem/commands/MoveGenericSubsystemWithPID.java

```diff
@@ -104,10 +80,10 @@ public void end(boolean interrupted) {
     public boolean isFinished() {
         double currentTime = Timer.getFPGATimestamp();
 
-        if(!pidController.atSetpoint()) {
+        if (!pidController.atSetpoint()) {
```

Comment:
Convention A-II\a2: Every pair of parentheses is followed by a single space and preceded by none